run.py

```diff
@@ -1,4 +1,4 @@
 from ros.app import app
 
 if __name__ == "__main__":
-    app.run(host='0.0.0.0', port=8080)
+    app.run(host='0.0.0.0', port=8082)
```

Comment:
We use the same run.py inside Dockerfile in `CMD` Also, 3scale service redirects the request to port 8080. 
I think we should consider changing the port of ingress container in docker-compose. I have raised the PR here - [https://github.com/RedHatInsights/ros-backend/pull/21/files](https://github.com/RedHatInsights/ros-backend/pull/21/files)
But this PR is a better fit for those changes.

Reply:
Thank you, I'll either integrate changes from that PR here, or just wait for them to be merged.

Reply:
+1 to make those changes here. As this PR is more relevant for those changes.